mycroft/configuration/__init__.py

```diff
@@ -13,4 +13,5 @@
 # limitations under the License.
 #
 from .config import Configuration, LocalConf, RemoteConf
+from .lang import set_default_lf_lang
```

Comment:
This should not be imported here if it is not used here.

Reply:
As above - the intention was to define the `mycroft.configuration` API but happy to put it wherever depending on the outcome of that thread.